src/main/java/com/sngular/kloadgen/sampler/KafkaProducerSampler.java

```diff
@@ -100,23 +100,29 @@ public void setupTest(final JavaSamplerContext context) {
       props2.putIfAbsent(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, AvroKafkaSerializer.class.getName());
 
       // Configure Service Registry location
-      props2.putIfAbsent(SerdeConfig.REGISTRY_URL, "http://localhost:8080");
+      //      props2.putIfAbsent(SchemaRegistryKeyHelper.SCHEMA_REGISTRY_URL, props.get(SchemaRegistryKeyHelper.SCHEMA_REGISTRY_URL));
```

Comment:
Question: why we have added this commented code?

Reply:
Those comments are going to remove